package.json

```diff
@@ -1,64 +1,15 @@
 {
-  "name": "@clappr/player",
-  "version": "0.5.0",
-  "description": "An extensible media player for the web",
-  "main": "dist/clappr.js",
-  "scripts": {
-    "build": "rollup --config",
-    "bundle-check": "ANALYZE_BUNDLE=true rollup --config",
-    "commitizen": "git-cz",
-    "lint": "eslint *.js src/",
-    "lint:fix": "npm run lint -- --fix",
-    "release": "MINIMIZE=true rollup --config",
-    "start": "DEV=true rollup --config --watch",
-    "test": "echo 'No tests configured. This project is only for bundling the player components.'",
-    "watch": "rollup --config --watch"
-  },
-  "files": [
-    "/dist",
-    "/src"
+  "name": "clappr-project",
```

Comment:
Here isn't good consider maintain the name clappr?

Reply:
We've agreed to keep the `clappr-project` name for the monorepo